#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_files_locations.dpatch by Daniel Leidert (dale) <daniel.leidert@wgdd.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: elements.xml and gtkperiodic.glade belong to the libgchemutils0 package
## DP: so we need to fix their location.

@DPATCH@
diff -urNad gnome-chemistry-utils-0.4.1/gcu/element.cc /tmp/dpep.xCDDGc/gnome-chemistry-utils-0.4.1/gcu/element.cc
--- gnome-chemistry-utils-0.4.1/gcu/element.cc	2004-09-17 14:37:47.000000000 +0200
+++ /tmp/dpep.xCDDGc/gnome-chemistry-utils-0.4.1/gcu/element.cc	2005-03-21 20:31:02.752390008 +0100
@@ -68,7 +68,7 @@
 	char *lang = getenv ("LANG");
 	char *old_num_locale, *tmp, *num;
 	unsigned char Z;
-	if (!(xml = xmlParseFile (DATADIR"/gchemutils/elements.xml")))
+	if (!(xml = xmlParseFile (DATADIR"/libgchemutils0/elements.xml")))
 	{
 		g_error (_("Can't find and read elements.xml"));
 	}
diff -urNad gnome-chemistry-utils-0.4.1/gcu/gtkperiodic.c /tmp/dpep.xCDDGc/gnome-chemistry-utils-0.4.1/gcu/gtkperiodic.c
--- gnome-chemistry-utils-0.4.1/gcu/gtkperiodic.c	2004-09-17 14:37:47.000000000 +0200
+++ /tmp/dpep.xCDDGc/gnome-chemistry-utils-0.4.1/gcu/gtkperiodic.c	2005-03-21 20:30:35.266568488 +0100
@@ -163,7 +163,7 @@
 	int i;
 	char* domain = g_strdup(textdomain(NULL));
 	textdomain(GETTEXT_PACKAGE);
-	xml =  glade_xml_new(DATADIR"/gchemutils/glade/gtkperiodic.glade", "vbox1", NULL);
+	xml =  glade_xml_new(DATADIR"/libgchemutils0/glade/gtkperiodic.glade", "vbox1", NULL);
 	g_return_if_fail (xml);
 	g_object_set_data(G_OBJECT(periodic), "xml", xml);
 	glade_xml_signal_autoconnect (xml);
